master/internal/db/postgres_rp_workspace_bindings_intg_test.go

```diff
@@ -34,10 +35,53 @@ func TestListWorkspacesBindingRP(t *testing.T) {
 	return
 }
 
-func TestListRPsBoundToWorkspace(t *testing.T) {
+func TestListRPsAvailableToWorkspace(t *testing.T) {
 	// pretty straightforward
 	// don't list binding that are invalid
 	// return unbound pools too (we pull from config)
+	require.NoError(t, etc.SetRootPath(RootFromDB))
+	db := MustResolveTestPostgres(t)
+	MustMigrateTestPostgres(t, db, MigrationsFromDB)
+	ctx := context.Background()
+	user := RequireMockUser(t, db)
+
+	// A workspace has no bounded RP
+	// TODO: Write RP config to master config?
```

Comment:
For testing, RP config doesn't need to be saved into master config. We can pass `existingPools` into functions.